README.md

```diff
@@ -3,104 +3,20 @@
 Run your own node.
 
 ## Installation
-### Latest release
-
-Download from the [releases page](https://github.com/AudiusProject/audius-d/releases), **OR** run the following:
-
-```bash
-curl -L "https://github.com/AudiusProject/audius-d/releases/latest/download/audius-ctl-$(uname -m)" -o ~/.local/bin/audius-ctl && chmod +x ~/.local/bin/audius-ctl
-```
-
-### From build
 
 ```bash
-make
-
-make install  # installs to ~/.local/bin/
-
-## OR ##
-
-sudo make install  # installs to /usr/local/bin/
+curl -sSL https://install.audius.org | sh
 ```
 
-### Uninstall
+#### Uninstall
 
 ```bash
 sudo make uninstall
 ```
```

Comment:
This uninstall only works if the repo is checked out, unlike the above install command